Soccar [Game-Server]/src/nl/soccar/gameserver/controller/rmi/GameServerRmiController.java

```diff
@@ -0,0 +1,102 @@
+package nl.soccar.gameserver.controller.rmi;
+
+import nl.soccar.library.SessionData;
+import nl.soccar.rmi.RmiConstants;
+import nl.soccar.rmi.interfaces.IMainServerForGameServer;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.rmi.NotBoundException;
+import java.rmi.RemoteException;
+import java.rmi.registry.LocateRegistry;
+import java.rmi.registry.Registry;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @author Lesley
+ */
+public final class GameServerRmiController {
+
+    private static final Logger LOGGER = Logger.getLogger(GameServerRmiController.class.getSimpleName());
+    private final String host;
+    private final int port;
+    private Registry registry;
+    private IMainServerForGameServer mainServerForGameServer;
+    private GameServerForMainServer gameServerForMainServer;
+
+    public GameServerRmiController(String host, int port) {
+        this.host = host;
+        this.port = port;
+    }
+
+    public void initialize() throws IOException {
+        registry = LocateRegistry.getRegistry(host, port);
+    }
+
+    public void bind() throws NotBoundException, RemoteException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.rmi.NotBoundException, java.rmi.RemoteException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1160)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)